stock_picking_report_by_partner/README.rst

```diff
@@ -0,0 +1,39 @@
+Picking Report by Partner
+=========================
+
+This module was written to extend the functionality of picking reports to support diferent picking report by partner and allow you to set the picking report template in partner form to print this template as default.
+
```

Comment:
You should note that the template is used only when you are clicking on the print button on the picking form, not when you click on meny Print > Picking List